.kiro/specs/auterity-ai-platform-unified/templates/preview-api-specification.py

```
@@ -0,0 +1,748 @@
+            except ImportError:
+                pass  # Module not available
+        
+        return safe_globals
+    
+    async def _validate_code_safety(self, code: str) -> None:
+        """Validate code for security issues"""
+        
+        # Check for blocked functions
+        for blocked_func in self.config.blocked_functions:
+            if blocked_func in code:
+                raise SecurityError(f"Blocked function '{blocked_func}' found in code")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)